Content.Server/_Goobstation/Devil/GameTicking/Rules/DevilRuleSystem.cs

```diff
@@ -0,0 +1,96 @@
+// SPDX-FileCopyrightText: 2025 GoobBot <uristmchands@proton.me>
+// SPDX-FileCopyrightText: 2025 Solstice <solsticeofthewinter@gmail.com>
+// SPDX-FileCopyrightText: 2025 SolsticeOfTheWinter <solsticeofthewinter@gmail.com>
+//
+// SPDX-License-Identifier: AGPL-3.0-or-later
+
+using System.Text;
+using Content.Server._Goobstation.Devil.Roles;
+using Content.Shared._Goobstation.Devil;
+using Content.Server.Antag;
+using Content.Server.GameTicking.Rules;
+using Content.Server.Mind;
+using Content.Server.Objectives;
+using Content.Server.Roles;
+using Content.Shared.NPC.Prototypes;
+using Content.Shared.NPC.Systems;
+using Content.Shared.Roles;
+using Robust.Shared.Audio;
+using Robust.Shared.Prototypes;
+
+namespace Content.Server._Goobstation.Devil.GameTicking.Rules;
```

Comment:
Unused using directives
```suggestion
using Content.Server.Roles;
using Content.Shared.NPC.Systems;

namespace Content.Server._Goobstation.Devil.GameTicking.Rules;
```